ccd-definition/AuthorisationCaseEvent/AuthorisationCaseEvent-HWF-R2-CUI-nonprod.json

```diff
@@ -0,0 +1,15 @@
+[
+  {
+    "CaseTypeID": "CIVIL",
+    "CaseEventID": "NO_REMISSION_HWF",
+    "AccessControl": [
+      {
+        "UserRoles": [
+          "caseworker-civil-admin",
+          "cui-admin-profile"
```

Comment:
I think you need to give access to  "caseworker-civil" also. plz check
Does "cui-admin-profile" need access, as per story it should be case worker only? Applicable to all places where "cui-admin-profile" or any other role except  "caseworker-civil-admin", "caseworker-civil", is used.

Reply:
added only cui-admin-profile has mentioned